k-distribution/tests/regression-new/cell-sort/test.k

```diff
@@ -19,12 +19,12 @@ module TEST
                   <k color="green"> $PGM:Pgm </k>
                   <functions color="blue">
                     <nextFunction> 0:Int </nextFunction> // It works again if we move this outside of <functions>
```

Comment:
Should I move it outside the function so the test would work?

Reply:
I think something changed. The test is now passing and in order to keep the changes I had to make before, I had to change `1.test.out`